README.md

```diff
@@ -1,29 +1,92 @@
-# Tuples 
+Learn Tuples with Swift
+======================
 
+We've come a long way to learn the fundamentals of Swift. With our knowledge of variables, `Strings`, `Booleans` and collections, we can now create *functions*.  Functions are the building blocks of any application - large or small. To introduce the new (and funny-sounding) idea of `tuples`, let's review how functions can be used to send and receive data.
 
-## Learning Objectives (Students will be able to ....) 
-- Describe a tuple as being a type that groups multiple values into a single compound value.
-- Explain that values within a tuple can be of any type and do not have to be of the same type as each other.
-- Iterate over the key-value pairs in a dictionary where each item is returned as a (key, value) tuple.
-- Create a tuple and name the individual elements
 
-## What the Student Already Knows
-A student should already understand the concept of variables, constants, Int, String, Bool, Array, Dictionary and functions but not Tuples.
+A Game of Chance
+----------------
+ Suppose you were creating a game that included the use of dice. You will randomly roll a pair a dice to receive some prize or to advance in the game. With our current knowledge of functions, the actions to start the game may look something like this:
 
```

Comment:
This is great. One of the things we like to focus on is creating a narrative that flows throughout the entire readme or lab. I like the idea of challenging the student early on to think.. that way it sticks with them as they read on. Asking the question here "So we are tasked to create a function that will generate two random numbers, how can we do this?" Tuples!